lfucg-exactions/server/accounts/viewsets.py

```diff
@@ -135,11 +137,13 @@ def create(self, request):
         serializer = PaymentSerializer(data=data_set)
         if serializer.is_valid(raise_exception=True):
             self.perform_create(serializer)
+            send_email_to_supervisors(entry = serializer.data, group = ['Finance'], perm_name = 'add_payment')
```

Comment:
seems to me it might be better to use signals to detect adds/changes to the models and send the emails. Have you looked into using signals instead?

Reply:
I have no idea what signals are but will look into those.